### ISSUE TITLE
[Bug] v.distance does not upload values to the attribute table

### ISSUE THREAD

**Issue by veroandreo** [OSGeo/grass#812]
v.distance yields this error `ERROR: At least one of the following options is required: <output>, <table> and <-p>` when trying to reproduce the example **Point-in-polygon** from the manual page. 

I have a vector of labeled points and a vector of polygons (output of i.segment) and I want to transfer the label I manually added to points to the attribute table of the segments for which I already extracted statistics to use it as train data for a classification using v.class.mlR. For some reason v.what.vect does not transfer all labels, hence I wanted to try with v.distance as suggested in the former's manual. 
 
```
g.copy vector=firestations,my_firestations
g.copy vector=zipcodes,my_zipcodes
v.db.addcolumn my_zipcodes column="ID_f integer"
v.distance from=my_zipcodes to=my_firestations upload=to_attr column=ID_f to_column=ID
```

**System description**
GRASS version: 7.9.dev                                                          
Code revision: c3a3f57aa                                                        
Build date: 2020-07-13                                                          
Build platform: x86_64-pc-linux-gnu                                             
GDAL: 3.0.4                                                                     
PROJ: 6.3.2                                                                     
GEOS: 3.8.1                                                                     
SQLite: 3.32.3                                                                  
Python: 3.8.3                                                                   
wxPython: 4.0.7                                                                 
Platform: Linux-5.7.8-200.fc32.x86_64-x86_64-with-glibc2.2.5

**Comment by neteler** [comment]
Probably this change in master is the reason:

https://github.com/OSGeo/grass/commit/2c614dd3225294a1f9e094b06d24a0905a239333

probably (I didn't check in details):
https://github.com/OSGeo/grass/blob/855f674e407a3bfa4b28c880f1cb0f14b24443c9/vector/v.distance/main.c#L269

**Comment by veroandreo** [comment]
Just for the record, v.what.vect does not work either in grass dev since it uses v.distance...

**Comment by veroandreo** [comment]
Fixed in 63eeced, thanks @mlennert :)
